catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -102,6 +102,8 @@
 
     public static final String CACHED = "cached";
 
+    private static final String INTERNAL_LOCAL_RESOURCE = "internal.local-resource";
```

Comment:
Ok, talked to @figliold and I got it now. I'm not sure we should blindly add all the "internal" attributes to the response. At least not as part of this ticket. There's a few things we would need to consider first. For instance, can we add _all_ attributes not in the metacard type or do some of them need to be hidden? Are we ok with the names we're using internally in the returned JSON?